docs/src/main/asciidoc/core.adoc

```diff
@@ -1,7 +1,7 @@
 [#spring-cloud-gcp-core]
 == Spring Cloud GCP Core
 
-At the center of every Spring Cloud GCP module are the concepts of `GcpProjectIdProvider` and `CredentialsProvider`.
+Each Spring Cloud GCP module uses `GcpProjectIdProvider` and `CredentialsProvider` to get the GCP project ID and credentials to access the Google Cloud.
```

Comment:
"to get the GCP project ID and access credentials"? Otherwise this sentence has both "GCP" and "Google Cloud", and looks like project ID is needed to access.

Reply:
done